mycelo/genesis/base_config.go

```diff
@@ -26,12 +26,10 @@ func BaseConfig() *Config {
 			TargetDensity:   fixed("0.5"),
 		},
 		Reserve: ReserveParameters{
-			TobinTaxStalenessThreshold: bigInt(3153600000),
-			TobinTax:                   common.Big0,
-			TobinTaxReserveRatio:       common.Big0,
-			DailySpendingRatio:         bigIntStr("50000000000000000000000"),
-			FrozenDays:                 nil,
-			FrozenGold:                 nil,
+			TobinTaxStalenessThreshold: 3153600000,
+			TobinTax:                   fixed("0"),
+			TobinTaxReserveRatio:       fixed("0"),
+			DailySpendingRatio:         fixed("0.05"),
```

Comment:
are all this `fixidity` numbers on the smarts contracts?

Reply:
They are, though I will check to make sure I'm understanding correctly why in some of them there is a `wrap` and in others not. 
 
https://github.com/celo-org/celo-monorepo/blob/0564826758fc171a36bf4131093e52773fad5254/packages/protocol/contracts/stability/Reserve.sol#L100-L101 and 
https://github.com/celo-org/celo-monorepo/blob/0564826758fc171a36bf4131093e52773fad5254/packages/protocol/contracts/stability/Reserve.sol#L158